### PR TITLE
Added images to cache

### LINKED ISSUE #202: Images do not work in offline mode on production
Images seem not to work on the production in offline mode. 
I think we should change current logic and introduce lazy loading with a service worker.

### PR DESCRIPTION
**Pull request by timuric**
Resolves #202

### CODE REVIEW

**Comment by maarcingebala** [comment]
@piotrgrundas Would you have a minute to review this PR?

File: src/components/CachedImage/CachedImage.tsx [hunk 1]
```diff
@@ -42,6 +42,10 @@ class CachedImage extends React.Component<CachedImageProps, CachedImageState> {
   }
 
   componentDidMount() {
+    if("caches" in window) {
+      const { url, url2x } = this.props;
+      window.caches.open('image-cache').then((cache)=> void cache.addAll([ url, url2x ]));
```
Comment (piotrgrundas) [inline src/components/CachedImage/CachedImage.tsx]: Do wee need void here? Also could u palce it in a separate method please. And space is missing after `(cache)`

File: src/components/CachedImage/CachedImage.tsx [hunk 2]
```diff
@@ -41,7 +41,15 @@ class CachedImage extends React.Component<CachedImageProps, CachedImageState> {
     }
   }
 
+  addImagesToCache(){
+    if("caches" in window) {
```
Comment (maarcingebala) [inline src/components/CachedImage/CachedImage.tsx]: We use [prettier](https://prettier.io/) for code formatting. Could you apply it to your changes?

Comment (timuric) [reply src/components/CachedImage/CachedImage.tsx]: @maarcingebala Actually I just wanted to ask about it. I don't see any configs or prettier commands in the project. Are you using default?

Comment (maarcingebala) [reply src/components/CachedImage/CachedImage.tsx]: Yep, I already spoke with @piotrgrundas that `prettier` should come with the project in `package.json`. Also, we could add it to CI to automatically fail PRs that introduce unformatted code. We're using the default config.

Comment (piotrgrundas) [reply src/components/CachedImage/CachedImage.tsx]: ATM just use prettier whit default settings and auto formatting on save.

**Pull request merged**